docs/vsphere-golden-image.md

```diff
@@ -4,13 +4,16 @@ This guide describes the thought process of creating a Windows virtual machine b
 
 ## 1. Select a compatible Windows Server version
 
-Currently, the Windows Machine Config Operator (WMCO) stable version only supports Windows Server Semi-Annual
-Channel (SAC): Windows Server 2004, that includes the patch [KB4565351](https://support.microsoft.com/en-us/help/4565351/windows-10-update-kb4565351), 
-required by the operating system to allow using the [hybrid OVN Kubernetes networking with a 
-custom VXLAN port](setup-hybrid-OVNKubernetes-cluster.md#vSphere) feature.
+Currently, the Windows Machine Config Operator (WMCO) stable version supports Windows Server Long-Term Servicing Channel
+(LTSC): Windows Server 2022 and Windows Server Semi-Annual Channel (SAC): Windows Server 20H2.
```

Comment:
nit: the presentation of these versions could make it easy to skim through
consider using bullet points to have the versions take up more space and draw more attention
i.e:



Currently, the Windows Machine Config Operator (WMCO) stable version supports:
* Windows Server 2022
* Windows Server 20H2

Reply:
Additionally the `Long-Term Servicing Channel` /`SAC` information isn't really needed IMO, just adds more noise